tests/flow/test_list.py

```
@@ -133,7 +133,7 @@ def test03_null_list_function_inputs(self):
         try:
             self.graph.query(query)
             assert(False)
-        except redis.exceptions.ResponseError:
+        except redis.ResponseError:
```

Ruff (0.15.7)

[error] `redis` may be undefined, or defined from star imports

(F405)